Update uk.m3u

Sphinxroot opened the pull request:
Add Some Channel More to come

Sphinxroot committed:
Update uk.m3u

Sphinxroot commented:
removed http://photocall.tv/  because 
ex:
http://photocall.tv/lF4r/ftv.m3u8?ch=film4   this part  lF4r  change everyday

Sphinxroot committed:
Update uk.m3u

LaneSh4d0w commented on channels/uk.m3u:
```diff
@@ -236,6 +260,14 @@ http://5.255.90.184:2001/play/a05b
 https://discoverfilm-discoverfilm-1-gb.samsung.wurl.com/manifest/playlist.m3u8
+#EXTINF:-1 tvg-id="DisneyXD.uk" tvg-name="Disney XD" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/SoWP2SK.png" group-title=""DISNEY XD
+http://stream.tvtap.live:8081/live/disneyxd.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="DiscoveryDMAX.uk" tvg-name="DMAX" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/15UWNF0.jpg" group-title="",Dmax
```
DMCAd channel, can't be added here.

Sphinxroot replied:
disney or dmax ?

MapGuy11 replied:
dmax

LaneSh4d0w replied:
DMAX

LaneSh4d0w commented on channels/uk.m3u:
```diff
@@ -297,6 +325,10 @@ https://brandusa-gousa-1-gb.samsung.wurl.com/manifest/playlist.m3u8
 https://brandusa-gousa-1-in.samsung.wurl.com/manifest/playlist.m3u8
 #EXTINF:-1 tvg-id="GodTVUK.uk" tvg-name="God TV UK" tvg-country="UK" tvg-language="English" tvg-logo="" group-title="Religious",God TV UK (720p)
 https://zypelive-lh.akamaihd.net/i/default_1@745545/master.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="UK" tvg-language="Greek" tvg-logo="https://i.imgur.com/YhwbcwE.png" group-title=""GREEK TV LONDON
```
```suggestion
#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="UK" tvg-language="Greek" tvg-logo="https://i.imgur.com/YhwbcwE.png" group-title="",GREEK TV LONDON
```

Sphinxroot replied:
corrected :D

LaneSh4d0w commented on channels/uk.m3u:
```diff
@@ -15,6 +19,10 @@ http://akaal.zecast.net/akaal-live/smil:akaaltv.smil/playlist.m3u8
 https://mn-nl.mncdn.com/alhiwar_live/smil:alhiwar.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="Alibi.uk" tvg-name="Alibi" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/IlXokSN.png" group-title="Series",Alibi
+http://stream.tvtap.live:8081/live/alibi.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="AMC.uk" tvg-name="AMC" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/7wKiXrs.jpg" group-title="Entertainment",AMC
```
Originates from the US.

Sphinxroot replied:
https://alibi.uktv.co.uk/ maybe let it here ?

LaneSh4d0w replied:
Okay... I also accept this one, but it's streamed in Europe : https://en.wikipedia.org/wiki/AMC_(European_TV_channel)

freearhey replied:
Better to move the channel to the American playlist, just leaving `tvg-country="UK"`.

LaneSh4d0w commented on channels/uk.m3u:
```diff
@@ -210,6 +222,16 @@ https://vs-hls-pushb-uk-live.akamaized.net/x=3/i=urn:bbc:pips:service:cbbc_hd/t=
 https://vs-hls-pushb-uk-live.akamaized.net/x=3/i=urn:bbc:pips:service:cbeebies_hd/mobile_wifi_main_sd_abr_v2_akamai_hls_live_http.m3u8
 #EXTINF:-1 tvg-id="CBeebiesHD.uk" tvg-name="CBeebies HD" tvg-country="UK" tvg-language="English" tvg-logo="" group-title="",CBeebies HD (720p) [Geo-Blocked]
 https://vs-hls-pushb-uk-live.akamaized.net/x=3/i=urn:bbc:pips:service:cbeebies_hd/t=3840/v=pv14/b=5070016/main.m3u8
+#EXTINF:-1 tvg-id="CBSDrama.uk" tvg-name="CBS Drama" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/fcsOntA.png" group-title="",CBS Drama
```
Originates from the US.

Sphinxroot replied:
https://en.wikipedia.org/wiki/CBS_Drama

MapGuy11 replied:
CBS Drama is owned by CBS UK Division.

freearhey replied:
Better to move the channel to the American playlist, just leaving `tvg-country="UK"`.

LaneSh4d0w commented on channels/uk.m3u:
```diff
@@ -210,6 +222,16 @@ https://vs-hls-pushb-uk-live.akamaized.net/x=3/i=urn:bbc:pips:service:cbbc_hd/t=
 #EXTINF:-1 tvg-id="CBeebiesHD.uk" tvg-name="CBeebies HD" tvg-country="UK" tvg-language="English" tvg-logo="" group-title="",CBeebies HD (720p) [Geo-Blocked]
 https://vs-hls-pushb-uk-live.akamaized.net/x=3/i=urn:bbc:pips:service:cbeebies_hd/t=3840/v=pv14/b=5070016/main.m3u8
+#EXTINF:-1 tvg-id="CBSDrama.uk" tvg-name="CBS Drama" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/fcsOntA.png" group-title="",CBS Drama
+http://stream.tvtap.live:8081/live/cbsdrama.stream/playlist.m3u8
```
Originates from the US.

MapGuy11 replied:
CBS Drama is owned by CBS UK Division.

freearhey replied:
Better to move the channel to the American playlist, just leaving `tvg-country="UK"`.

LaneSh4d0w commented on channels/uk.m3u:
```diff
@@ -210,6 +222,16 @@ https://vs-hls-pushb-uk-live.akamaized.net/x=3/i=urn:bbc:pips:service:cbbc_hd/t=
 https://vs-hls-pushb-uk-live.akamaized.net/x=3/i=urn:bbc:pips:service:cbeebies_hd/t=3840/v=pv14/b=5070016/main.m3u8
+#EXTINF:-1 tvg-id="CBSDrama.uk" tvg-name="CBS Drama" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/fcsOntA.png" group-title="",CBS Drama
+http://stream.tvtap.live:8081/live/cbsdrama.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="CBSReality.uk" tvg-name="CBS Reality" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/k28RZK2.jpg" group-title="",CBS Reality
```
Originates from the US.

Sphinxroot replied:
https://en.wikipedia.org/wiki/CBS_Reality

MapGuy11 replied:
CBS Reality is owned by CBS UK Division.

freearhey replied:
Better to move the channel to the American playlist, just leaving `tvg-country="UK"`.

LaneSh4d0w commented on channels/uk.m3u:
```diff
@@ -236,6 +260,14 @@ http://5.255.90.184:2001/play/a05b
 http://5.255.90.184:2001/play/a05h
 #EXTINF:-1 tvg-id="DiscoverFilm.uk" tvg-name="Discover.Film" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/yjIYUim.jpg" group-title="",Discover.Film (720p)
 https://discoverfilm-discoverfilm-1-gb.samsung.wurl.com/manifest/playlist.m3u8
+#EXTINF:-1 tvg-id="DisneyXD.uk" tvg-name="Disney XD" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/SoWP2SK.png" group-title=""DISNEY XD
```
Originates from the US.

LaneSh4d0w replied:
That one closed down

LaneSh4d0w commented on channels/uk.m3u:
```diff
@@ -377,6 +413,12 @@ https://ooyalahd2-f.akamaihd.net/i/mtach7_delivery@348438/master.m3u8
 https://ooyalahd2-f.akamaihd.net/i/mtaengaudio_delivery@138280/master.m3u8
 #EXTINF:-1 tvg-id="nTVUSEastern.uk" tvg-name="n TV (US Eastern)" tvg-country="UK" tvg-language="" tvg-logo="https://i.imgur.com/l75bDTx.png" group-title="",n TV (US Eastern)
 https://a.jsrdn.com/broadcast/22680_3BR3zocwi9/-0500/c.m3u8
+#EXTINF:-1 tvg-id="Nick.uk" tvg-name="Nick" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/6riwdSg.png" group-title="Kids",Nick
```
Originates from the US.

LaneSh4d0w replied:
https://en.wikipedia.org/wiki/Nickelodeon_(British_and_Irish_TV_channel)

freearhey replied:
Better to move the channel to the American playlist, just leaving `tvg-country="UK"`.

LaneSh4d0w commented on channels/uk.m3u:
```diff
@@ -377,6 +413,12 @@ https://ooyalahd2-f.akamaihd.net/i/mtach7_delivery@348438/master.m3u8
 https://a.jsrdn.com/broadcast/22680_3BR3zocwi9/-0500/c.m3u8
+#EXTINF:-1 tvg-id="Nick.uk" tvg-name="Nick" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/6riwdSg.png" group-title="Kids",Nick
+http://stream.tvtap.live:8081/live/nick.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="NickJunior.uk" tvg-name="Nick Jr" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/G3e8rWz.jpg" group-title="Kids",Nick Jr
```
Originates from the US. (pretty sure we have it too)

LaneSh4d0w replied:
https://en.wikipedia.org/wiki/Nick_Jr._(British_and_Irish_TV_channel)

I'm starting to think there's always a British-specific version of the US channels.

freearhey replied:
Better to move the channel to the American playlist, just leaving `tvg-country="UK"`.

LaneSh4d0w commented on channels/uk.m3u:
```diff
@@ -391,6 +433,18 @@ https://qwestclassic-samsung-uk.amagi.tv/playlist.m3u8
 #EXTINF:-1 tvg-id="QUESTTVMIX.uk" tvg-name="QUEST TV MIX" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/B69BHnx.jpg" group-title="",QUEST TV MIX (720p)
 https://qwestmix-samsung-uk.amagi.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="QVC.uk" tvg-name="QVC" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/q8jkrYW.png" group-title="Shop",QVC
+https://d1txbbj1u9asam.cloudfront.net/live/qvcuk_main_clean/bitrate1.isml/3/prog_index.m3u8
```
Originates from the US. (pretty sure we have it too)

LaneSh4d0w replied:
Accepting it seems QVC had made an arrangement with Sky to create their own QVC, and is streamed from the UK : https://gaz.wiki/wiki/fi/QVC_(British_TV_channel)

freearhey replied:
Better to move the channel to the American playlist, just leaving `tvg-country="UK"`.

LaneSh4d0w commented on channels/uk.m3u:
```diff
@@ -391,6 +433,18 @@ https://qwestclassic-samsung-uk.amagi.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="QVC.uk" tvg-name="QVC" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/q8jkrYW.png" group-title="Shop",QVC
+https://d1txbbj1u9asam.cloudfront.net/live/qvcuk_main_clean/bitrate1.isml/3/prog_index.m3u8
+#EXTINF:-1 tvg-id="QVC.uk" tvg-name="QVC" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/q8jkrYW.png" group-title="Shop",QVC +1
+https://live-qvcuk.simplestreamcdn.com/hera/remote/qvcuk_primary_sdi1/live.m3u8
```
Originates from the US. (pretty sure we have it too)

Sphinxroot replied:
https://www.qvcuk.com/content/shop-live-tv.html

LaneSh4d0w replied:
Accepting it seems QVC had made an arrangement with Sky to create their own QVC, and is streamed from the UK : https://gaz.wiki/wiki/fi/QVC_(British_TV_channel)

freearhey replied:
Better to move the channel to the American playlist, just leaving `tvg-country="UK"`.

LaneSh4d0w commented on channels/uk.m3u:
```diff
@@ -391,6 +433,18 @@ https://qwestclassic-samsung-uk.amagi.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="QVC.uk" tvg-name="QVC" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/q8jkrYW.png" group-title="Shop",QVC +1
+https://live-qvcuk.simplestreamcdn.com/hera/remote/qvcuk_primary_sdi1/live.m3u8
+#EXTINF:-1 tvg-id="QVCBeauty.uk" tvg-name="QVC Beauty" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/Xb2t2EF.jpg" group-title="Shop",QVC Beauty
+http://live.qvcuk.simplestreamcdn.com/live/qvcuk_beauty_clean/bitrate1.isml/live.m3u8
```
Originates from the US.

Sphinxroot replied:
https://www.qvcuk.com/content/shop-live-tv.html

LaneSh4d0w replied:
Accepting it seems QVC had made an arrangement with Sky to create their own QVC, and is streamed from the UK : https://gaz.wiki/wiki/fi/QVC_(British_TV_channel)

freearhey replied:
Better to move the channel to the American playlist, just leaving `tvg-country="UK"`.

LaneSh4d0w commented on channels/uk.m3u:
```diff
@@ -391,6 +433,18 @@ https://qwestclassic-samsung-uk.amagi.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="QVCBeauty.uk" tvg-name="QVC Beauty" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/Xb2t2EF.jpg" group-title="Shop",QVC Beauty
+http://live.qvcuk.simplestreamcdn.com/live/qvcuk_beauty_clean/bitrate1.isml/live.m3u8
+#EXTINF:-1 tvg-id="QVCExtra.uk" tvg-name="QVC Extra" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/r8UP0bE.png" group-title="Shop",QVC Extra
+https://live-qvcuk.simplestreamcdn.com/live/qvcuk_extra_clean/bitrate1.isml/live.m3u8
```
Originates from the US. (pretty sure we have it too)

Sphinxroot replied:
https://www.qvcuk.com/content/shop-live-tv.html

LaneSh4d0w replied:
QVC in its essence was born in the US.

MapGuy11 replied:
If the link is for a uk specific channel it goes in uk.m3u not us.m3u as the channel would not be in the US.

LaneSh4d0w replied:
Accepting it seems QVC had made an arrangement with Sky to create their own QVC, and is streamed from the UK : https://gaz.wiki/wiki/fi/QVC_(British_TV_channel)

freearhey replied:
Better to move the channel to the American playlist, just leaving `tvg-country="UK"`.

LaneSh4d0w commented on channels/uk.m3u:
```diff
@@ -391,6 +433,18 @@ https://qwestclassic-samsung-uk.amagi.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="QVCExtra.uk" tvg-name="QVC Extra" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/r8UP0bE.png" group-title="Shop",QVC Extra
+https://live-qvcuk.simplestreamcdn.com/live/qvcuk_extra_clean/bitrate1.isml/live.m3u8
+#EXTINF:-1 tvg-id="QVCStyle.uk" tvg-name="QVC Style" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/T8jwQJh.jpg" group-title="Shop",QVC Style
+http://live.qvcuk.simplestreamcdn.com/live/qvcuk_style_clean/bitrate1.isml/live.m3u8
```
Originates from the US.

Sphinxroot replied:
https://www.qvcuk.com/content/shop-live-tv.html all those channel have UK curency a bit confusing

freearhey replied:
Better to move the channel to the American playlist, just leaving `tvg-country="UK"`.

LaneSh4d0w commented on channels/uk.m3u:
```diff
@@ -455,6 +529,8 @@ https://csm-e-stv.tls1.yospace.com/csm/live/139900483.m3u8
 #EXTINF:-1 tvg-id="SupertoonsTV.uk" tvg-name="Supertoons TV" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/35BpWJh.jpg" group-title="Kids",Supertoons TV (720p)
 https://kedoo-supertoonstv-1-gb.samsung.wurl.com/manifest/playlist.m3u8
+#EXTINF:-1 tvg-id="Syfy.uk" tvg-name="Syfy" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/CDeBomC.png" group-title="",SYFY
+http://stream.tvtap.live:8081/live/uk-syfy.stream/playlist.m3u8
```
Originates from the US.

Sphinxroot replied:
exemple   ads  are intended for uk https://i.imgur.com/iP0Lq8w.png  so it get confusing where to put them

MapGuy11 replied:
If the link is for a uk specific channel it goes in uk.m3u not us.m3u as the channel would not be shown/streamed in the US.

LaneSh4d0w replied:
SyFy is a US-based channel, but according to https://en.wikipedia.org/wiki/Syfy_(British_and_Irish_TV_channel), this one is UK-based and a totally different Syfy.

freearhey replied:
Better to move the channel to the American playlist, just leaving `tvg-country="UK"`.

LaneSh4d0w commented on channels/uk.m3u:
```diff
@@ -473,6 +549,8 @@ https://tennischannel-intl-samsung-uk.amagi.tv/playlist.m3u8
 #EXTINF:-1 tvg-id="TJC.uk" tvg-name="TJC" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/cFlNhjV.png" group-title="",TJC (1080p)
 https://cdn-shop-lc-01.akamaized.net/Content/HLS_HLS/Live/channel(TJCOTT)/index.m3u8
+#EXTINF:-1 tvg-id="Tnt.uk" tvg-name="Tnt" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/CzuwTbL.png" group-title="",TNT
+http://stream.tvtap.live:8081/live/uk-tnt.stream/playlist.m3u8
```
Originates from the US.

MapGuy11 replied:
If the link is for a uk specific channel it goes in uk.m3u not us.m3u as the channel would not be shown/streamed in the US.

LaneSh4d0w replied:
https://en.wikipedia.org/wiki/TNT_(American_TV_network)

Sphinxroot committed:
Update uk.m3u

Sphinxroot committed:
Update uk.m3u

Sphinxroot reviewed (commented):
maybe we can let  those channel intended  for UK viewer  most of them have UK English  and not  international or US English

MapGuy11 committed:
Merge branch 'master' into Sphinxroot-patch-1

LaneSh4d0w commented:
The idea that those channels were US-based and that there were actually UK variants hosted in the UK made me confused. Approving the majority except TNT, in which I haven't found a UK variant.

MapGuy11 commented:
> The idea that those channels were US-based and that there were actually UK variants hosted in the UK made me confused. Approving the majority except TNT, in which I haven't found a UK variant.

Oh Ok :D

Sphinxroot committed:
Update uk.m3u

removed TNT And Dmax thanks for the clarification  sometime it get confusuing where to put stuff   , some channel are sorted by country hosting , TV network , and it not sorted for target viewer  since some TV channel have many language for them but they get in the same playlist

freearhey reviewed (changes requested):
The whole point of sorting channels by file is just to avoid duplicate links and the more compact we store them, the easier it is to control. When generating other playlists, only what is written in the `tvg-country` is taken into account.

BellezaEmporium committed:
Merge branch 'master' into Sphinxroot-patch-1
